libcontainer/seccomp/types.go

```diff
@@ -0,0 +1,48 @@
+package seccomp
```

Comment:
I have a few issues with the current API, but nothing I would consider major.
* No ability to set default action - much harder to make a whitelist, you basically have to list every syscall. Should be simple to add to the Seccomp struct.
* I think representing syscalls by name, and not number, is superior. Architecture independence and clarity, mainly. Libseccomp gives us the ability to easily do architecture-independent resolution.
* I don't think a one-op masked equality operator makes much sense --- might as well use the libseccomp convention of two operands per syscall argument.